BitwardenShared/UI/Auth/Extensions/AuthRouter+Redirects.swift

```diff
@@ -208,7 +208,8 @@ extension AuthRouter {
             // If no account can be set to active, go to the landing or carousel screen.
             let isCarouselEnabled: Bool = await services.configService.getFeatureFlag(.nativeCarouselFlow)
             let introCarouselShown = await services.stateService.getIntroCarouselShown()
-            return isCarouselEnabled && !introCarouselShown ? .introCarousel : .landing
+            let shouldShowCarousel = isCarouselEnabled && !introCarouselShown
+            return shouldShowCarousel && !isInAppExtension ? .introCarousel : .landing
```

Comment:
⛏️ I believe we could move the condition check into the `shouldShowCarousel` line given that is one of the conditions for that to happen so the last line is simpler and all the conditions are in one place.
```suggestion
            let shouldShowCarousel = isCarouselEnabled && !introCarouselShown && !isInAppExtension
            return shouldShowCarousel ? .introCarousel : .landing
```

Reply:
🤔 I kind of feel like it's six of one, half dozen of the other. But putting all the conditions in one line is reasonable.